test/integration/stomp_/publisher/create_request_test.py

```diff
@@ -130,7 +139,7 @@ def test_listen_create_request(self):
 
     @pytest.mark.usefixtures("easy_client", "request_generator")
     def test_publish_create_request(self):
-        """Published the Request over STOMP and verifies of HTTP"""
+        """Publish a Request over STOMP and verify it via HTTP."""
```

Comment:
I know this is very nitpicky, but the pythonic way to write doc strings is for the first line to begin with a verb in the imperative mood. `pydocstyle` enforces this. We're not using it, but maybe we should (?)

Reply:
I'm generally in favor of abdicating all styling decisions to some tool that will enforce it for us.  So I'm down with `pydocstyle`, though at first glance theres...a lot of work ahead of us to become compliant.